tests/db_engine_specs/hive_tests.py

```diff
@@ -16,6 +16,10 @@
 # under the License.
 from unittest import mock
 
+from pyhive import hive
+from sqlalchemy import column
+from sqlalchemy.dialects import *
```

Comment:
Hi,
is there a specific reason to have an import star? Otherwise, I personally would prefer single imports :)

Reply:
+1, let's not `import *` as we can't trace where objects are coming from, otherwise LGTM

Reply:
Is this import really needed? It seems none of the standard Sql Alchemy dialects are used here.

Reply:
Right, and that very question proves the point of how hard it is to track what's behind the `*`